en/docs/manage-databases-and-caches/choreo-managed-postgresql-databases.md

```diff
@@ -6,7 +6,7 @@ PostgreSQL on Choreo offers fully Choreo-managed, efficient object-relational da
 
 Follow the steps below to create a Choreo-managed PostgreSQL database: 
 
-1. From the environment list on the header, located next to the **Deployment Tracks** list, select your **Organization**.
+1. From the organization list on the header, located next to the **Deployment Tracks** list, select your **Organization**.
```

Comment:
We don't need to say "located next to the **Deployment Tracks** list" part right?
Shall we go with the following steps:
1. Sign in to the Choreo Console at https://console.choreo.dev/.
2. In the header, click the **Organization** list. This opens the organization home page.
3. In the left navigation menu, click **Dependencies** and then **Databases**.
<continue the rest of the steps>